### ISSUE TITLE
show command label instead of command self in the pipeline-step

### ISSUE THREAD

**Issue by mPokornyETM** [jenkinsci/pipeline-graph-view-plugin#144]
### What feature do you want to see added?

We use in many cases use the option *label* for system commands. It is great feature to inform the user about 'what the step do'
But no body cares about the system command self.

Like in my example it will be great, when only 'print node info' will be shown.

![image](https://user-images.githubusercontent.com/89339813/193457614-8b00bbba-5ab8-4ca1-88ce-571ab7c5af04.png)

```

stage('Node info') {
    node() {
      Map opts = [label : 'print node info', script : 'hostname']
      if (isUnix()) {
          sh opts
      } else {
          bat opts
      }
      deleteDir();
    }
}
```

Maybe it shall be configurable (somewhere global, per user or some user matrix)

**Comment by timja** [comment]
Could you clarify what you mean by `self`? I'm guessing you mean don't show the shell script if label is set?

**Comment by mPokornyETM** [comment]
> Could you clarify what you mean by `self`? I'm guessing you mean don't show the shell script if label is set?

Yes you are right. I use *self* because it can be *sh* or *bat* command.

**Comment by mPokornyETM** [comment]
May be it will be enough when the order is chagne
```
label - shell
```
this is not so big deal.

**Comment by stuartrowe** [comment]
@mPokornyETM is this resolved for you after https://github.com/jenkinsci/pipeline-graph-view-plugin/pull/164?

**Comment by mPokornyETM** [comment]
> @mPokornyETM is this resolved for you after #164?

yes it works as expected, thx
![image](https://user-images.githubusercontent.com/89339813/236168400-cd952db2-6167-4c3f-a0bf-078a99eb1639.png)
